src/Castle.Facilities.AutoTx.Tests/Castle.Facilities.AutoTx.Tests.csproj

```diff
@@ -34,8 +34,10 @@
     <PackageReference Include="Castle.LoggingFacility" Version="4.1.0" />
     <PackageReference Include="Castle.Windsor" Version="4.1.0" />
     <PackageReference Include="NLog" Version="4.5.4" />
-    <PackageReference Include="NUnit" Version="2.6.4" />
+    <PackageReference Include="NUnit" Version="3.10.1" />
     <PackageReference Include="SharpTestsEx.MVC" Version="1.1.1" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.10.0" />

```

Comment:
Need the adapter to make tests discoverable/debuggable in Visual Studio v15.7.3. Resharper and the internal test runner.